components/status_dropdown/status_dropdown.test.tsx

```diff
@@ -169,7 +169,7 @@ describe('components/StatusDropdown', () => {
             <StatusDropdown {...props}/>,
         );
 
-        expect(wrapper.find('.status-dropdown-menu__clear-container').exists()).toBe(true);
+        expect(wrapper.find('.status-dropdown-menu__clear-container').exists()).toBe(false);
```

Comment:
Does this snapshot need to be updated? As far as i can see the UI shouldn't change here

Reply:
Okay,  I got it. I will look into it.